_posts/en/newsletters/2020-03-11-newsletter.md

```diff
@@ -0,0 +1,123 @@
+---
+title: 'Bitcoin Optech Newsletter #88'
+permalink: /en/newsletters/2020/03/11/
+name: 2020-03-11-newsletter
+slug: 2020-03-11-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to a description of methods for preventing
+hardware wallets from leaking private information through transaction
+signatures and provides an update on the BIP322 generic signmessage
+protocol.  Also included are our regular sections about new releases and
+notable merges of popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Exfiltration resistant nonce protocols:** Pieter Wuille sent an
+  [email][wuille overview] to the Bitcoin-Dev mailing list providing an
+  overview of techniques that can prevent a hardware wallet or other
+  offline signing device from communicating secret information to a
+  third party by biasing the ECDSA or schnorr signatures it creates.
+  The email is clearly written and packed with information.  Anyone with
+  an interest in secure use of external signers should consider reading
+  it.
+
+- **BIP322 generic signmessage---progress or perish:** [BIP322][] author
+  Karl-Johan Alm noted that his PR to add support for the [generic
+  signmessage protocol][topic generic signmessage] has not see any
+  progress towards getting merged for the past several months.  He's
+  [seeking feedback][alm feedback]---including "unfiltered
+  criticism"---about whether to take a different approach or to simply
+  abandon the proposal.  As we've mentioned [previously][segwit
+  signmessage], there is currently no widely adopted way for wallets to
+  create and verify signed messages for anything besides legacy P2PKH
+  addresses.  If wallet developers want to enable this capability for
+  P2SH, P2WPKH, P2WSH, and (if taproot is activated) P2TR addresses,
+  they are recommended to review Alm's email and provide feedback on
+  their preferred path forward.
+
+## Bitcoin Core PR Review Club
+
+The [Bitcoin Core PR Review Club][] is a weekly IRC meeting for newer
+contributors to the Bitcoin Core project to learn about the review process. An
+experienced Bitcoin Core contributor provides background information about a
+selected PR and then leads a discussion on IRC.
+
+The Review Club is an excellent way to learn about the Bitcoin protocol, the
+Bitcoin Core reference implementation, and the process of making changes to
+Bitcoin. Notes, questions and meeting logs are posted on the website for those
+who are unable to attend in real time, and as a permanent resource for those
+wishing to learn about the Bitcoin Core development process.
+
+In this section, we summarise a recent Review Club meeting.
+
+- **[Try to preserve outbound block-relay-only connections during restart][review club 17428]:**
```

Comment:
I think it's great this section is being added. But have some suggestions for formatting. I find it strange that ~80% of the content is in (), and hard to mentally parse. 

My suggestion- 
```
Discussion began by establishing..
-> Q: A
-> Q: A

Then changes in the PR were analyzed 
-> Q: A
-> Q: A
```

I think the breaks in pacing will make it easier to mentally parse.